scio-parquet/src/main/scala/com/spotify/scio/parquet/types/ParquetTypeIO.scala

```diff
@@ -109,9 +107,11 @@ object ParquetTypeIO {
   object ReadParam {
     private[types] val DefaultPredicate = null
     private[types] val DefaultConfiguration = new Configuration()
+    private[types] val DefaultSkipClone = true
   }
   final case class ReadParam[T] private (
     predicate: FilterPredicate = null,
+    skipClone: Boolean = ReadParam.DefaultSkipClone,
```

Comment:
I would put the `skipClone` param at the end of the params list, after `conf`! (same for `def typedParquetFile` below)

Reply:
Done 👍